_topic_maps/_topic_map.yml

```
@@ -3576,19 +3576,24 @@ Topics:
   Topics:
   - Name: Distributed tracing architecture
     File: distr-tracing-architecture
-- Name: Distributed tracing installation
-  Dir: distr_tracing_install
-  Topics:
-  - Name: Installing distributed tracing
-    File: distr-tracing-installing
-  - Name: Configuring the distributed tracing platform
-    File: distr-tracing-deploying-jaeger
-  - Name: Configuring distributed tracing data collection
-    File: distr-tracing-deploying-otel
-  - Name: Upgrading distributed tracing
-    File: distr-tracing-updating
-  - Name: Removing distributed tracing
-    File: distr-tracing-removing
+- Name: Distributed tracing platform - Jaeger
+  Dir: distr_tracing_jaeger
+  Topics:
+    - Name: Installing Jaeger
+      File: distr-tracing-jaeger-installing
+    - Name: Configuring Jaeger
+      File: distr-tracing-jaeger-configuring
+    - Name: Updating Jaeger
+      File: distr-tracing-jaeger-updating
+    - Name: Removing Jaeger
+      File: distr-tracing-jaeger-removing
+- Name: Distributed tracing data collection - OpenTelemetry
```

I noticed that the section about [the CLI tools uses parentheses for the tool name in the nav entry](https://docs.openshift.com/container-platform/4.13/cli_reference/index.html), so you could adopt the same convention:
```suggestion
- Name: Distributed tracing data collection (OpenTelemetry)
```